package-manifest.toml

```diff
@@ -148,20 +127,25 @@ switch_variant = "stub"
 #   1c. cargo xtask dist -o
 # 2. Copy dendrite.tar.gz from dendrite/out to omicron/out
 # 3. Use type = "manual" instead of the "prebuilt"
-type = "prebuilt"
-repo = "dendrite"
-commit = "09c8878be3f78f0a5d384e7f59b93ea66e02e7a4"
-sha256 = "e05d33dc5d3deafae7d618c45b019c87ddd97e7a1fff11fa3ffb7593a3e10eaf"
+source.type = "prebuilt"
+source.repo = "dendrite"
+source.commit = "09c8878be3f78f0a5d384e7f59b93ea66e02e7a4"
+source.sha256 = "e05d33dc5d3deafae7d618c45b019c87ddd97e7a1fff11fa3ffb7593a3e10eaf"
+output.type = "zone"
 
-[external_package.dendrite-asic]
-service_name = "switch"
-zone = true
-[external_package.dendrite-asic.only_for_targets]
-switch_variant = "asic"
-[external_package.dendrite-asic.source]
-# Manually overriding the package source is identical to dendrite-stub,
-# but built with "--features=tofino_asic"
-type = "prebuilt"
-repo = "dendrite"
-commit = "09c8878be3f78f0a5d384e7f59b93ea66e02e7a4"
-sha256 = "208ae10a61f834608378eb135e4b6e5993dc363019b8fba75465b6ea5506b635"
+[package.dendrite-asic]
+service_name = "dendrite"
+only_for_targets.switch_variant = "asic"
+# To manually override the package source:
+#
+# 1. Build the zone image manually
+#   1a. cd <dendrite tree>
+#   1b. cargo build --features=<tofino_stub|tofino_asic|softnpu>
+#   1c. cargo xtask dist -o
+# 2. Copy the output zone image from dendrite/out to omicron/out
+# 3. Use type = "manual" instead of the "prebuilt"
```

Comment:
```suggestion
# 3. Use source.type = "manual" instead of "prebuilt"
```

Reply:
Done in 3b469a56d646e2d827d6b880c0db84e0b65537c1